app/Repositories/Eloquents/PermissionEloquentRepository.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: quantien
+ * Date: 11/8/18
+ * Time: 8:52 AM
+ */
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Namespaces.NamespaceDeclaration.BlankLineAfter</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/PermissionEloquentRepository.php</br>**Line**: 9</br>There must be one blank line after the namespace declaration